app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __len__(self):
+        return self.km
+
+    def __truediv__(self, other):
+        if isinstance(other, Distance):
+            return Distance(km=round(self.km / other.km, 2))
+        elif isinstance(other, (int, float)):
+            return Distance(km=round(self.km / other, 2))
+
+    def __mul__(self, other):
+        if isinstance(other, Distance):
+            return Distance(km=self.km * other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(km=self.km * other)
```

Comment:
`other` can't be an instance of `Distance`, only a number (float or int)
Because if you multiply a kilometer by a kilometer, you get a square kilometer